Project.toml

```diff
@@ -10,7 +10,18 @@ HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JSONSchema = "7d188eb4-7ad8-530c-ae41-71a32a6d4692"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-julia = "1.0"
+julia = ">=1.0"
+DataStructures = ">= 0.15.0"
+GZip = ">= 0.5.0"
+HTTP = ">= 0.8.2"
+JSON = ">= 0.20.0"
+JSONSchema = ">= 0.1.1"
+MathOptInterface = ">= 0.8"
```

Comment:
We should put an upper bound on MOI. It will likely break soon.

Reply:
`MathOptInterface = "0.8, 0.9"` then?

Reply:
I'm not sure about the new syntax. Following https://julialang.github.io/Pkg.jl/v1/compatibility/#Tilde-specifiers-1
I think
`MathOptInterface = "~0.8"`

Reply:
I applied this one
https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1

Reply:
I read `"0.8, 0.9"` to mean `union("^0.8", "^0.9")`, so `[0.8, 0.10)`.

Reply:
ok, I thought this would break after 0.9. Then it is "~0.8.0" as you mentioned.

Reply:
addressed in d2b7652